app/views/oneOffContributions.scala.html

```diff
@@ -0,0 +1,24 @@
+@import assets.AssetsResolver
+@import com.gu.identity.play.IdUser
+@import config.StripeConfig
+@import helper.CSRF
+@(
+        title: String,
+        id: String,
+        js: String,
+        isTestUser: Boolean,
+        stripeConfig: StripeConfig
+)(implicit assets: AssetsResolver)
+
+@scripts = {
+    <script type="text/javascript">
+        window.guardian = window.guardian || {};
+        window.guardian.stripeKey = "@stripeConfig.publicKey";
```

Comment:
We need to use a different stripe key for one-off contributions, it's a different stripe account.